src/theory/quantifiers/bv_inverter.cpp

```diff
@@ -18,11 +18,30 @@
 #include <stack>
 
 #include "theory/quantifiers/term_database.h"
+#include "theory/bv/theory_bv_utils.h"
 
 using namespace CVC4;
 using namespace CVC4::kind;
+using namespace CVC4::theory;
 using namespace CVC4::theory::quantifiers;
 
+/* Drop child at given index from expression.
+ * E.g., dropChild((x + y + z), 1) -> (x + z)  */
+static Node dropChild (Node n, unsigned index)
+{
+  unsigned nchildren = n.getNumChildren();
+  Assert (index < nchildren);
+  Kind k = n.getKind();
+  Assert (k == AND || k == OR || k == BITVECTOR_MULT || k == BITVECTOR_PLUS);
+  NodeBuilder<> nb (NodeManager::currentNM(), k);
+  for (size_t i = 0, nchildren = n.getNumChildren(); i < nchildren; ++i)
+  {
+    if (i == index) continue;
```

Comment:
Why is `i` of type `size_t` and `index` of type `unsigned`?

Reply:
No particular reason. Will change size_t to unsigned.